public/style.css

```diff
@@ -0,0 +1,25 @@
+#panel-container {
+  margin-top: 5%;
+  width: 50%;
+}
+
+#panel-body {
```

Comment:
minor change here, ids should be unique, in this case we have a `panel-body` for each pet, so you just need to change these to a class selector: `.panel-body`, since you are already giving them that class